resources/client/functions.ts

```diff
@@ -0,0 +1,213 @@
+import { Delay } from '../utils/fivem';
+
+let prop = 0;
+let isPhoneOpen = false;
+let propCreated = false;
+let phoneModel = 'prop_amb_phone'; // Refered to in newphoneProp function. Requires custom phone being streamed.
```

Comment:
Not sure why this was a let in the first place but lets change it while we are here

```suggestion
const phoneModel = 'prop_amb_phone'
```